docs/components/table.md

```diff
@@ -243,6 +243,7 @@ object supporting the following properties:
 - Pagination support:
   - `page`: the currently visible page - starts at `1`
   - `pageSize`: the number of items to display per page
+  - `offset`: the value of the offset for offset-based pagination. `offset` **MUST** be included in the fetcher params for offset-based pagination to work properly.
```

Comment:
These doc updates need to also be copied over to KCatalog

Reply:
We don't currently have a use case for KCatalog, so I haven't added support for using offset-based pagination to KCatalog yet.